src/main/cs/Infinispan/HotRod/Impl/RemoteCacheSWIGImpl.cs

```diff
@@ -468,7 +468,7 @@ public void AddClientListener(Event.ClientListener<K,V> cl, string[] filterFacto
                         case (byte)EventType.CLIENT_CACHE_FAILOVER:
                             {
                                 Event.ClientCacheFailoverEvent ev = new Event.ClientCacheFailoverEvent();
```

Comment:
This "ev" is unused now. It seems a little strange that the failover events are handled differently from all the other events. But that was probably a goal?